script/shaders.ps1

```diff
@@ -0,0 +1,89 @@
+param
+(
+[Parameter(Mandatory=$false)]
+[string]$InputDir = "" + $(get-location).Path + "\resources\shaders",
+
+[Parameter(Mandatory=$false)]
+[string]$OutputDir = "" + $(get-location).Path + "\resources\shaders\compiled"
+)
+
+$Compiler = $Env:VULKAN_SDK + "\Bin\glslangValidator.exe"
+$Optimizer = $Env:VULKAN_SDK + "\Bin\spirv-opt.exe"
+$Preamble = "#pragma pack_matrix(row_major)"
+$TempDir = Join-Path $OutputDir "temp"
+
+$FailureCount = 0
+
+if (-not (Test-Path -Path $TempDir)) {
+    New-Item -Path $TempDir -ItemType Directory | Out-Null
+}
+
+Write-Host "--Compiling Shaders--"
+Get-ChildItem -Path $InputDir | Where-Object { $_.Extension -eq '.psh' -or $_.Extension -eq '.vsh' } |
+ForEach-Object {
+    if ($_.Extension -eq '.psh') {
+        $Stage = 'frag'
+        $FileName = "$($_.BaseName)Fragment.spv"
```

Comment:
It's an annoying change to make - but Diligent seems to use "Pixel" instead of "Fragment" everywhere. Can we change all uses of "Fragment" naming to "Pixel"? Then we match all throughout the codebase

Reply:
Done